### ISSUE TITLE
Suggestion: Remove original DS3 input after creating virtual input to prevent conflict/double taps

### ISSUE THREAD

**Issue by jarylc** [Emanem/js2xbox#1]
I'm not very familiar with C++ on Linux inputs as of yet and I can't really do a pull request on it. But if possible, could this be implemented?

It's something like --detech-kernel-driver for xboxdrv.

**Comment by Emanem** [comment]
Hi,

I haven't done what you suggested for 3 reasons:

1. If the input pad (i.e. PS3) is already visible by the application, then there is no point in using this utility
2. If the input pad is _not_ visible to the application, then why try to remove it (you can do easily with USB, not so easy with Bluetooth interface btw)? the application will only see the new virtual Xbox 360 pad anyway (example [The Valley](http://store.steampowered.com/app/378610/))...
3. It might require root privileges for something not necessary

**Comment by jarylc** [comment]
Hey Emanem

There are also another reasons to use this tool even though the PS3 controller can be used.

1) incorrect mappings on games that does not allow keybind changing (Valley)
2) using Steam Big Picture

Well these are pretty rare cases, if you feel it's not worth doing it, it's fine really.

But maybe it'll motivate me to tinker on the codes haha :P

**Comment by jarylc** [comment]
Oh I didn't read the Valley portion, I'm amazed you are playing the same game. Yeah I'm connected via BT usually.

If I'm not wrong, Steam Big Picture mode tries to map the PS3 controller to Xbox 360 controller on games too, so it's pretty much using Big Picture that is the cause of my problems.

**Comment by Emanem** [comment]
Right, never though about that (Big Picture).

Detaching a device I think can be *easily* done with USB devices (albeit you might need to be *root*). For Bluetooth devices, I'm afraid not sure how easy would be.
This is why I've decided to completely avoid doing it.

Another way I tried to implement this change (i.e. removing the devices) was using filesystem overlays, but then the exported filesystem wouldn't let the user access the event filesystem...

I would love to have a pull request from you btw, if you ever decide to code this.
